gemini/sample-apps/fashion-trends-prediction/app/genAIprompts.py

```diff
@@ -0,0 +1,175 @@
+
+Output:
+
+{{
+        "Color": {{
+            "Black": 16,
+            "Red": 4,
+            "White": 3,
+            "Brown": 1,
+            "Pink": 1
+        }},
+
+
+        "Length": {{
+            "mini": 4,
+            "maxi": 3
+        }},
+
+
+        "Sleeve Length": {{
+            "off-the-shoulder": 3,
+            "strapless": 3,
+            "one-shoulder": 3,
+            "long-sleeved": 1
+        }},
+
+
+        "Neckline": {{
+            "off-the-shoulder": 3,
+            "strapless": 3,
+            "one-shoulder": 3,
+            "plunging": 2,
+            "halter": 1
+        }},
+
+
+        "Material": {{
+            "sequin": 2,
+            "sheer": 2,
+            "leather": 1,
+            "fur": 1
+        }},
+
+
+        "Style": {{
+            "bodycon": 1,
+            "ball gown": 1,
+            "wrap": 1,
+            "asymmetrical": 1,
+            "see-through": 1
+        }},
+
+
+        "Pattern": {{
+            "Solid": 19,
+            "Striped": 1
+        }}
+}}
+
+###################
+Input:
+
+You have the following pants data -
+
+['black wide-legged pants', 'black pants', 'plaid pants', 'olive green pleated pants', 'red latex pants', 'black pants', 'black pants', 'black pants', 'white sweatpants', 'black pants with silver studs', 'blue denim pants with crystal embellishment', 'black baggy pants', 'red sweatpants', 'black pants', 'black pants', 'black pants', 'black pants', 'black pants', 'grey sweatpants', 'faded blue low-waisted baggy jeans', 'white pants', 'black ski pants', 'brown flare pants', 'black leather pants with knee pads', 'light blue jeans with stars']
+
+First think of the different aspects/features in a pants. Eg. color, print, etc.
+
+I want to know the maximum occurring values for each aspect. So, for each aspect, summarize the above data and list at most 5 values with counts in descending order of counts in json format.
+
+Output:
+
+{{
+        "Color": {{
+            "Black": 14,
+            "Blue": 3,
+            "White": 2,
+            "Red": 2,
+            "Grey": 1
+        }},
+
+
+        "Fit": {{
+            "sweatpants": 3,
+            "baggy": 2,
+            "wide-legged": 1,
+            "low-waisted": 1
+        }},
+
+
+        "Material": {{
+            "denim": 1,
+            "latex": 1,
+            "leather": 1,
+            "plaid": 1
+        }},
+
+
+        "Embellishment": {{
+            "studs": 1,
+            "crystal": 1,
+            "stars": 1
+        }}
+}}
+
+
+###################
+Input:
+
+You have the following {category} data -
+
+{data}
+
+First think of the different aspects/features in a {category}. Eg. color, print, etc.
+
+I want to know the maximum occurring values for each aspect. So, for each aspect, summarize the above data and list at most 5 values with counts in descending order of counts in json format.
+
+Output:
+
+
+"""
+
+
+qList = [
```

Comment:
Use snake case. And be more specific with the names of these variables.

Reply:
Removed as it was unused